Portal accelerator

tagolovina opened the pull request:
This PR fixes adds

1. Portal Accelerator Form json file
Breaking Changes
1. Added   "cuaidAMBAAccelerator": "dddb1f42-f9d8-48e3-9e6b-f1ce3e9c2c76" to  alzArm.json

arjenhuitema commented on patterns/alz/alz-portal.json:
```diff
@@ -0,0 +1,1090 @@
+  "view": {
+    "kind": "Form",
+    "properties": {
+      "title": "Azure Monitor Baseline Alerts Accelerator",
```
- [ ] Can you add "[Preview] -" to the title.
- [ ] Please add ALZ to the name for example "AMBA-ALZ Accelerator". To prevent any misunderstandings as this only deploys the ALZ-Pattern.

arjenhuitema commented on docs/content/patterns/alz/deploy/Deploy-via-Azure-Portal-UI.md:
```diff
@@ -0,0 +1,96 @@
+---
+title: Deploy via the Azure Portal UI
```
- [ ] Can you change the name to "Deploy via Azure Portal" ?

arjenhuitema reviewed (changes requested):
Love the Accelerator. Great work. I left a couple of comments for you to review.

Also, yesterday the accelerator worked perfectly. Currently getting an error message with [this URL](https://portal.azure.com/#view/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ftagolovina%2Fazure-monitor-baseline-alerts%2Fportal%2Fpatterns%2Falz%2FalzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Ftagolovina%2Fazure-monitor-baseline-alerts%2Fportal%2Fpatterns%2Falz%2Falz-portal.json), can you check?

tagolovina commented:
- Fixed documentation page
- Changed list of regions to API
- Changed Deployment Settings page in Accelerator according to comments about ALZ aligned
- Changed link to short aka.ms in documentation
